lib/Tags/Tags.js

```diff
@@ -28,6 +28,8 @@ class Tags extends React.Component {
 
   static propTypes = {
     children: PropTypes.func,
+    entityPath: PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)]),
```

Comment:
Just wondering - I don't have any stake in this - but might it be a bit safer to just have a `getEntity` prop instead, with a default value of `getEntity: props => props.resources?.entities?.records?.[0],`.

It wouldn't be a big complexity change for users of this override, but it:
- allows just a bit more functionality than a hardcoded path
- is a more common API (function) than an array of path tokens
- lets us minimize our usage of lodash/get

Hell, I'm not sure exactly how you're using `entityWrapper`, but it might let you get away with not needing that prop at all....

Reply:
It makes sense.
Will think about this.

Reply:
For the sake of completeness the default prop than should be 
```
getEntity: props => props.resources?.entities?.records?.[0] || {},
```